languages/erlang/config.json

```
@@ -5,7 +5,15 @@
     "indent_size": 4
   },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "basics",
+        "uuid": "e21279ba-ee09-412a-902d-478ccdc5d28c",
+        "prerequisites": [],
+        "topics": ["math", "functions", "variables"],
+        "difficulty": 1
+      }
+    ],
     "practice": []
```

With the changing spec, we've also introduced a new top-level object: `concepts`. All the concepts listed in the exercise's `concepts` key should have a corresponding entry in the `concepts` key. Taking into account the suggestion added in the above comment, this would mean adding:

```suggestion
    "practice": []
    "concepts": [
      {
        "uuid": "2eb4a463-355f-46ef-ac55-a75ec5afdf86",
        "slug": "basics",
        "name": "Basics",
        "blurb": "TODO: add blurb for basics concept"
    }
  ]
```

The `blurb` can be updated later,